opm/simulators/wells/GasLiftSingleWell_impl.hpp

```diff
@@ -136,24 +136,24 @@ computeWellRates_(Scalar bhp, bool bhp_is_limited, bool debug_output ) const
     return {-potentials[this->oil_pos_],
             -potentials[this->gas_pos_],
             -potentials[this->water_pos_],
+            bhp,
```

Comment:
This now returns `BasicRates` which seems to indicate to the reader that well rates are returned, but `BasicRates` now includes `bhp` which is a pressure.